src/components/Popover/Popover.js

```diff
@@ -46,12 +46,17 @@ class Popover extends Component {
     /**
      * alignment of the popover relative to the button
      */
-    align: alignPropType
+    align: alignPropType,
+    /**
+     * onClose callback functin
```

Comment:
You have a typo here (`function`) 😉
Could you make this more descriptive, please? Something like: `A function that is called when the Popover is closed"

Reply:
Thanks, changed some of the other descriptions as well :)